readthedocs/settings/base.py

```diff
@@ -245,15 +239,26 @@ def INSTALLED_APPS(self):  # noqa
     # All auth
     ACCOUNT_ADAPTER = 'readthedocs.core.adapters.AccountAdapter'
     ACCOUNT_EMAIL_REQUIRED = True
-    ACCOUNT_EMAIL_VERIFICATION = "mandatory"
-    ACCOUNT_AUTHENTICATION_METHOD = "username_email"
+    ACCOUNT_EMAIL_VERIFICATION = 'mandatory'
+    ACCOUNT_AUTHENTICATION_METHOD = 'username_email'
     ACCOUNT_ACTIVATION_DAYS = 7
-    SOCIALACCOUNT_EMAIL_VERIFICATION = "none"
+    SOCIALACCOUNT_EMAIL_VERIFICATION = 'none'
     SOCIALACCOUNT_AUTO_SIGNUP = False
     SOCIALACCOUNT_PROVIDERS = {
         'github': {
-            'SCOPE': ['user:email', 'read:org', 'admin:repo_hook', 'repo:status']
-        }
+            'SCOPE': [
+                'user:email',
+                'read:org',
+                'admin:repo_hook',
+                'repo:status',
+            ],
+        },
+        'gitlab': {
+            'SCOPE': [
+                'api',
+                'read_user',
```

Comment:
Not sure, but maybe the `sudo` permission is also required.

Reply:
We'll want to make sure here. Folks are really particular about the permissions we request

Reply:
I'm not sure to understand their documentation. Seems poor to me: https://docs.gitlab.com/ce/user/profile/personal_access_tokens.html#limiting-scopes-of-a-personal-access-token

So far so good, I was able to connect the account and setup the webhooks using only these two.